packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -43,12 +43,6 @@
 }
 
 .edit-site-page-patterns-dataviews {
-	margin-top: 60px;
```

Comment:
Instead of adding this bit of CSS to every page that displays the content frame in a mobile viewport, we move this to the hub component instead.